### PR TITLE
Update waas-servicing-channels-windows-10-updates.md, issue 835

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #835 clarification, 
source: 
1. [Windows 10 and the “disappearing” SAC-T](https://techcommunity.microsoft.com/t5/Windows-IT-Pro-Blog/Windows-10-and-the-disappearing-SAC-T/ba-p/199747)

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/update/waas-servicing-channels-windows-10-updates.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ ms.topic: article
 >
 >Due to [naming changes](waas-overview.md#naming-changes), older terms like CB, CBB and LTSB may still be displayed in some of our products.
 
-Semi-Annual Channel (Targeted) is the default servicing channel for all Windows 10 devices except those with the LTSB edition installed. The following table shows the servicing channels available to each edition of Windows 10. 
+Semi-Annual Channel is the default servicing channel for all Windows 10 devices except those with the LTSB edition installed. The following table shows the servicing channels available to each edition of Windows 10.
```
Comment (nenonix) [inline windows/deployment/update/waas-servicing-channels-windows-10-updates.md]: **The** Semi-Annual Channel **(SAC)** is the default servicing channel for all Windows 10 **devices,** except those with the LTSB edition installed. The following table shows the servicing channels available to **each Windows 10 edition**.

File: windows/deployment/update/waas-servicing-channels-windows-10-updates.md [hunk 2]
```diff
@@ -44,6 +44,9 @@ Semi-Annual Channel (Targeted) is the default servicing channel for all Windows
 >[!NOTE]
 >The LTSB edition of Windows 10 is only available through the [Microsoft Volume Licensing Center](https://www.microsoft.com/Licensing/servicecenter/default.aspx).
 
+>[!NOTE]
+>Semi-Annual Channel (Targeted) should be used only by the customers that are using [Windows Update for Business](https://docs.microsoft.com/windows/deployment/update/waas-manage-updates-wufb). For those, who doesn't use Windows Update for Business, Semi-Annual Channel (Targeted) would be the same as Semi-Annual Channel.
```
Comment (nenonix) [inline windows/deployment/update/waas-servicing-channels-windows-10-updates.md]: For those, who **don't** use Windows Update for Business, Semi-Annual Channel (Targeted) **is** the same as Semi-Annual Channel.

File: windows/deployment/update/waas-servicing-channels-windows-10-updates.md (hunk 1 above)
Comment (JohanFreelancer9) [reply windows/deployment/update/waas-servicing-channels-windows-10-updates.md]: Agree.

File: windows/deployment/update/waas-servicing-channels-windows-10-updates.md (hunk 2 above)
Comment (JohanFreelancer9) [reply windows/deployment/update/waas-servicing-channels-windows-10-updates.md]: Agree.

**Comment by MaratMussabekov** [comment]
Dear @nenonix ,
your edits were applied
Thank you

**Comment by mypil** [comment]
@jaimeo - can you please assist to check, approve and merge the content for this PR based on issue #835?

Thank you.

**Comment by mypil** [comment]
@jaimeo - I would like to follow up on this. Kindly assist to check, approve and merge the content for this PR based on issue #835.

Thank you.

**Comment by jaimeo** [comment]
We will be updating all topics that refer to Semi-Annual Channel (Targeted) for an upcoming release--most references will be removed; however, certain instances must remain for a little while. I have a SME who will need to double-check this particular proposed edit prior to acceptance--I'll ask her to have a quick look at this now.

**Comment by mypil** [comment]
@jaimeo - I would like to follow up. Do you have any updates on this?

**Comment by mypil** [comment]
@jaimeo - I would like to follow up on this. Kindly assist to check, approve and merge the content for this PR based on issue #835.

Thank you.

**Pull request merged**